README.md

```diff
@@ -31,6 +31,14 @@ In order to be able to use the library following environment properties have to
 | OIDC_CLIENT_SECRET_${idp} | Client Secret for the OIDC configuration. The Client Secret is a confidential value known only to the application and the OIDC provider, used to authenticate the application to the provider. |
 | CONTENT_CONFIGURATION_VALIDATOR_API_URL | Endpoint URL for custom content configuration validation. Endpoint returns useful error message when configuration is invalid.  |
 
+
+- **Portal**
+
+| Property name                               | Description                                                                                                                                                                                     |
+|---------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OPENMFP_PORTAL_CONTEXT_CRD_GATEWAY_API_URL  | This URL is used to retrieve Content Configurations form database                                                                                                                                                  |
```

Comment:
this is a portal specific env variable should n't be here, this section is wrong